packages/core/src/errorCollection.ts

```diff
@@ -165,24 +164,24 @@ export function toStackTraceString(stack: StackTrace) {
 export function trackNetworkError(
   configuration: Configuration,
   errorObservable: ErrorObservable,
-  requestObservable: RequestObservable
+  requestObservable: Observable<RequestCompleteEvent>
 ) {
-  requestObservable.subscribe((request: RequestDetails) => {
-    if (isRejected(request) || isServerError(request)) {
+  requestObservable.subscribe((completeEvent: RequestCompleteEvent) => {
```

Comment:
In all contexts where we use `completeEvent`, it would seem more appropriate to me to use `request`